### ISSUE TITLE
Seismogram (preview) plotting seems to be broken with matplotlib 1.5.0

### ISSUE THREAD

**Issue by petrrr** [obspy/obspy#1168]
Plotting of seismographs using the Trace.plot() and Stream.plot() method. Seems to be broken with matplotlib 1.5.0. At least for me this problem appeared with a recent update to 1.5.0 on OS X and a downgrade fixed the issue again. Observed with ObsPy version 0.10.2. I have not managed to test master yet, for another issue with matplotlib 1.5.0 in an virtualenv (*TODO: separated issue*)

Observation:
When invoking the method the following warning appear:
```
/opt/local/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/matplotlib/axes/_base.py:2767: UserWarning: Attempting to set identical left==right results
in singular transformations; automatically expanding.
left=733503.0, right=733503.0
  'left=%s, right=%s') % (left, right))
```
The plot shows a time window of several years, and it the center the seismogram is squeezed to a central line. When zooming onto the signal, it also seems that the time reference is shifted by some hours or truncated. The date is still okay.

**Comment by QuLogic** [comment]
This is fixed on `master` and `releases` by #1123.

**Comment by petrrr** [comment]
Sorry, was looking through the tickets, but missed this one.
